src/default-settings.js

```diff
@@ -8,6 +8,7 @@ const settings = {
     endy_col: 'AENDY',
     sev_col: 'AESEV',
     rel_col: 'AEREL',
+    ser_col: 'AESER',
```

Comment:
After reading through the rest of this and reviewing the discussion on #12, I do think we're breaking backwards compatibility. Not a huge deal, we'll just need to go through and update milestones and update package.json. Agree?